src/packages/v2v3/components/V2V3Project/ProjectDashboard/components/CyclesPayoutsPanel/hooks/useConfigurationPanel/useFormatConfigurationExtensionSection.ts

```diff
@@ -1,14 +1,14 @@
 import { t } from '@lingui/macro'
+import { flagPairToDatum } from 'components/Project/ProjectHeader/utils/flagPairToDatum'
```

Comment:
Is ProjectHeader the best place for this? 🤔

Reply:
That's my mistake, was meant to put it in ProjectTabs, moved it there now.